es-vue-base/src/lib-icons/states/icon-state-dc.vue

```diff
@@ -0,0 +1,43 @@
+            fill="currentColor"
```

Comment:
So DC is the only state graphic that had two colors before, per Maria's design, to highlight the DC part of Maryland. ([example rebates & incentives page](https://www.energysage.com/local-data/solar-rebates-incentives/dc/))

Can you check with Asheeta on how important she feels it is to keep those two colors? If so, maybe we default to a certain darker orange for the inner DC piece and accept a prop for that secondary color that can be passed in to customize that default?

<img width="331" alt="Screen Shot 2024-06-11 at 6 45 50 AM" src="https://github.com/EnergySage/es-ds/assets/1350363/46791161-88cc-43ea-ac98-92865dafee86">

Reply:
I assumed that was a mistake, and didn't even notice the DC cutout 😂. I'll check with Asheeta

Reply:
Ok, I've made it so the surrounding area (not DC) is brighter than the rest using a CSS filter. I figure this will be a saner default value than a hardcoded orange. The brightness is also adjustable by prop.